From be4e885d473ffe96bcdf52d2b431d490d26cb969 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 27 Jun 2022 14:28:56 +0200
Subject: [PATCH] Hack for Marvell switches

This is hack, which ensures that we have working Marvell switches
---
 kmod-mox.list   | 3 +++
 kmod-omnia.list | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/kmod-mox.list b/kmod-mox.list
index fe1381a..19b5028 100644
--- a/kmod-mox.list
+++ b/kmod-mox.list
@@ -7,3 +7,6 @@ gpio-button-hotplug
 mwlwifi
 mwifiex-sdio
 btmrvl
+
+# Hack
+dsa-mv88e6xxx
diff --git a/kmod-omnia.list b/kmod-omnia.list
index 8ae58d1..fccb6fd 100644
--- a/kmod-omnia.list
+++ b/kmod-omnia.list
@@ -8,3 +8,6 @@ eeprom-at24
 ledtrig-gpio
 ledtrig-oneshot
 ledtrig-transient
+
+# Hack
+dsa-mv88e6xxx
-- 
2.34.1

